### ISSUE TITLE
Switch to a libsqlite3-based Sqlite library

### ISSUE THREAD

**Issue by busyboredom** [busyboredom/acceptxmr#62]
The Arti TOR client uses Sqlite, and only sys crate for a given C lib can be used at once. To be compatible with Arti, the Sqlite storage implementation should be changed to use a libsqlite3-based Sqlite crate.

**Comment by CM-IV** [comment]
I could be wrong, but this might help with people already using Sqlite in their applications with the `sqlx` crate as well.  I tried this a bit ago and ran into an issue with the underlying C libs for Sqlite.

**Comment by busyboredom** [comment]
Yep, you are right. libsqlite3 is the de facto standard, so the current little-known alternative that AcceptXMR uses is causing compatibility issues in many places right now.
